logging/zerolog/client_interceptors.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 Michal Witkowski. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package grpc_zerolog
```

Comment:
Could we change the package name `grpc_zerolog` > `zerolog` we are going to have to do a large refactor / breaking change to fix this elsewhere :(